draft-ietf-tls-esni.md

```diff
@@ -686,8 +693,8 @@ an ECHConfig that can be used to successfully decrypt
 ClientEncryptedCH.encrypted_ch. This matching procedure should be done using
 one of the following two checks:
 
-1. Compare ClientEncryptedCH.config_digest against cryptographic hashes of known
-   ECHConfig and choose the one that matches.
+1. Compare ClientEncryptedCH.config_digest against digests of known ECHConfig
```

Comment:
Since this is no longer a hash function output (and not a digest), perhaps we should rename this to `config_id`, or something.

Reply:
I'd be ok with this change, though existing issues refer to `config_digest`. Why don't we put this on the stack of editorial TODOs?

Reply:
Yeah, that's fine.

Reply:
Noted.